### ISSUE TITLE
MQTT source - possibility to pass default number of partition a replication factor

### ISSUE THREAD

**Issue by dberardo-com** [lensesio/stream-reactor#1361]
seems like the mqtt source connector does not take into account:

 "topic.creation.default.partitions"        : 
 "topic.creation.default.replication.factor": 

https://rmoff.net/2021/01/06/creating-topics-with-kafka-connect/

**Comment by andrewstevenson** [comment]
@dberardo-com These are Connect Framework settings and not for the connector plugin to implement. Are you running Kafka 2.6 or above?

**Comment by dberardo-com** [comment]
kafka 3.x with KRAFT i find the kafka doc very hard to follow when it comes to kafka connect.

which parameters are those which are natively supported?
